.github/workflows/backup.yml

```diff
@@ -0,0 +1,12 @@
+name: Backup Postgres
+on: [workflow_dispatch]
```

Comment:
we can move this to a cron later - but we do want this to run every night

Reply:
to clarify I meant a github actions cron schedule
eg. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule